src/KafkaFlow/Consumers/IMessageConsumer.cs

```diff
@@ -49,21 +59,21 @@ public interface IMessageConsumer
         /// </remarks>
         string ClientInstanceName { get; }
 
-        /// <summary>
-        /// Gets the current number of workers allocated of the consumer
-        /// </summary>
-        int WorkerCount { get; }
-
         /// <summary>
         /// Gets the current consumer flow status
         /// </summary>
-        ConsumerFlowStatus FlowStatus { get; }
+        ConsumerFlowStatus? FlowStatus { get; }
```

Comment:
Why the change to nullable?

Reply:
Because when we are restarting the consumer, the `Consumer.InvalidateConsumer` method sets null into `FlowManager` property.
 
If during this time interval someone tries to retrieve information about this consumer (through the web api, for example) it would occur an exception.

To avoid this behaviour, the FlowManager now is nullable and the ConsumerResponseAdapter returns `ConsumerFlowStatus.NotRunning` status if consumer.FlowStatus.

Reply:
Another option would be to put this logic directly into `MessageConsumer`, keeping the `ConsumerFlowStatus` as not nullable and returning `ConsumerFlowStatus.NotRunning` when `FlowManager` is null.

Actually, I think it would be better this approach.... what do you think?

Reply:
We can move the ConsumerFlowStatus to Consumer class, calculate the NotRunning status based on FlowStatus is null or partitions assignment is empty or null.

Rename ConsumerFlowStatus to ConsumerStatus too